### ISSUE TITLE
[SpicesUpdate@claudiux] Shows Updated Theme Available No Obvious Way To Update 

### ISSUE THREAD

**Issue by Tater-K7S5A** [linuxmint/cinnamon-spices-applets#5017]
```
 * Applet version/Build date 6.4.2
 * Cinnamon version 5.8.4
 * Distribution - Mint 21.2
 * Graphics hardware *and* driver used  Radeon RX 570 + AMDGPU
 * Snippet or attachment of ~/.cinnamon/glass.log
 * Snippet or attachment of ~/.xsession-errors

 ```

**Notify author of applet**
@claudiux 

**Issue**
I have a theme I downloaded from https://cinnamon-spices.linuxmint.com ([Numix Cinnamon Transparent](https://cinnamon-spices.linuxmint.com/themes/view/Numix-Cinnamon-Transparent)) which has received an update.  SpicesUpdate applet has notified me of this fact.  Unfortunately, it fails to give me any indication of how to update the theme. Clicking on the 'Themes" entry opens the system Themes dialog which contains no mechanism that I can see to pull down an update.  

I know I can just download and extract the updated theme into my ~/.themes but I am sure that most users will be perplexed by the inconsistent behavior of SpicesUpdate.  

The Applets notification will take you to the system Applets dialog which allows you to download and update applets.  I have not used Desklets or Extensions but they appear to be the same as Applets.  Themes is the odd man out.

Since Cinnamon does not seem to have a built in way of updating themes and it is, obviously, outside the scope of SpicesUpdate to write an application to do that, I would suggest:

1)  A tool tip or note that manual download is required. Or,  

2) A link to https://cinnamon-spices.linuxmint.com/themes on the 'Themes' line of the SpicesUpdate Applet indicating that manual update is required.

Basically, something to help normal users understand what it is that they need to do to update their theme.

You are doing excellent work here and I really appreciate your efforts and commitment.  Thank you!

**Steps to reproduce**

**Expected behaviour**

**Other information**

**Comment by Tater-K7S5A** [comment]
I have been directed to the 'Add/Remove' tab and now see the update mechanism.  

Closed:  Headspace and timing error.
